src/libraries/libraries-packages.proj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.Build.Traversal" DefaultTargets="Pack">
 
   <PropertyGroup>
-    <TraversalGlobalProperties>BuildAllProjects=true</TraversalGlobalProperties>
```

Comment:
do we no longer need BuildAllProjects with the new version ?

Reply:
The libraries packages don't need the BuildAllProjects global property.